README.md

```diff
@@ -35,6 +35,22 @@ EventMachine.run do
 end
 ```
 
+To pass additional parameters while subscribing, add them to the identifier hash. These will be available server side in the channel's params hash.
```

Comment:
As far as I know rails doesn't automatically extract query parameters into the params hash from websocket requests.